.github/workflows/bats-action.yml

```diff
@@ -64,14 +64,17 @@ jobs:
            file: ./tests/Dockerfile.rpi4-BATS
            builder: ${{ steps.buildx.outputs.name }}
            tags: localhost:5000/openhabian/rpi4-openhabian:latest
+           cache-from: type=gha
+           cache-to: type=gha,mode=max
            push: true
        - name: BATS tests on virtual 64bit RPi4 running Debian Bullseye
          run: |
-           docker run --name "openhabian-rpi4" -d localhost:5000/openhabian/rpi4-openhabian
+           docker run --rm --name "openhabian-rpi4" -d localhost:5000/openhabian/rpi4-openhabian
            docker exec -i "openhabian-rpi4" bash -c 'bats --tap --recursive --filter "development-." .'
            docker exec -i "openhabian-rpi4" bash -c 'bats --tap --recursive --filter "unit-." .'
            docker exec -i "openhabian-rpi4" bash -c 'bats --tap --recursive --filter "installation-." .'
            docker exec -i "openhabian-rpi4" bash -c 'bats --tap --recursive --filter "destructive-." .'
+           docker stop "openhabian-rpi4"
          shell: bash
   amd64-bats:
     runs-on: ubuntu-latest
```

Comment:
So if I correctly understood you replace the Ubuntu CI run by a generic Debian bullseye one.
Would you also need to replace the "ubuntu" in "runs-on" ?

Reply:
No, that is just the system that GH Actions is running on. Docker is running on a different system.